app/models/user.rb

```diff
@@ -265,6 +265,10 @@ def vso_employee?
     roles.include?("VSO")
   end
 
+  def can_view_vlj?
```

Comment:
```suggestion
  def cannot_view_vlj?
```
I think this will be the opposite. Also, it will need to include those with the `VSO` role as well.